hydra/controllers/client.py

```diff
@@ -816,3 +816,58 @@ def disable_metrics(self):
 
         self.restart_service()
         self.app.log.info(f'Successfully disabled all metrics reporting.')
+
+    @ex(
+        arguments=[
+            (
+                ['-n', '--name'],
+                {
+                    'help': 'name of network for key backup',
+                    'action': 'store',
+                    'dest': 'name'
+                }
+            ),
+            (
+                ['--d', '--destination'],
+                {
+                    'help': 'directory to which keys are backed up',
+                    'action': 'store',
+                    'dest': 'destination'
+                }
+            ),
+            (
+                ['-D', '--destroy'],
+                {
+                    'help': 'Use to overwrite existing backups',
+                    'action': 'store_true',
+                    'dest': 'destroy'
+                }
+            ),
+        ]
+    )
+    def backup_nodekeys(self, destroy=False):
+        name = self.app.utils.env_or_arg(
+            'name', 'HYDRA_NETWORK', or_path='.hydra_network', required=True)
+
+        filenames = [f'{name}/chaindata/config/node_key.json',
+                     f'{name}/chaindata/config/priv_validator.json']
+
+        destination = self.app.utils.path(
+            self.app.pargs.destination) if self.app.pargs.destination else self.app.utils.path(f'~/.hydra/{name}')
+
+        if os.path.exists(destination):
+            if not self.app.pargs.destroy:
+                self.app.log.warning('A backup already exists for {name}, use -D to overwrite.')
+                return
+            rmtree(destination)
+
+        os.makedirs(destination)
+
+        for filename in filenames:
+            self.app.log.info(f'Backing up key from {filename} to {destination}')
+            file = self.app.utils.path(destination + '/' + filename.split('/')[-1])
```

Comment:
I still think we should go with this approach: https://github.com/ShipChain/hydra/pull/27#discussion_r292020482

We always want the files in the network directory structure to be backed-up and restored as-is. The use case is that the user would want to be backing up _both_ their loom.yaml config in the network folder as well as the config data in the chaindata/config folder.

Reply:
Got it. I think this should now work. I'm using project_path = os,.getcwd() to get the default project path but there is probably a better way to do that.